test/fixtures/tracks/fake/config.json

```diff
@@ -2,6 +2,7 @@
   "slug": "fake",
   "language": "Fake",
   "repository": "https://github.com/exercism/xfake",
+  "checklist_issue": "1",
```

Comment:
Oh, also I think this should be a number, not a string, unless you have a good reason why it shouldn't be.